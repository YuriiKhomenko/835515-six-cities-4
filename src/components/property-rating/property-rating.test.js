import React from "react";
import renderer from "react-test-renderer";
import PropertyRating from "../property-rating/property-rating.jsx";

const rating = 5;

it(`Render Property Rating component`, () => {
  const tree = renderer
    .create(<PropertyRating
      rating={rating}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
