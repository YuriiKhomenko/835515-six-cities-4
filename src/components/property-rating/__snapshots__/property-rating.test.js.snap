// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Property Rating component 1`] = `
<div
  className="property__rating rating"
>
  <div
    className="property__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "100%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
  <span
    className="property__rating-value rating__value"
  >
    5
  </span>
</div>
`;
